src/atc.coffee

```diff
@@ -41,6 +41,13 @@ module.exports = (robot) ->
   validateTarget = (msg, {applicationName, environmentName}) ->
     validateApplicationName(msg, applicationName) && validateEnvironmentName(msg, applicationName, environmentName)
 
+  robot.respond /application list/i, (msg) ->
+    applications = robot.brain.data.applications
+    if applications.length > 0
+      msg.reply "applications available: #{applications.sort().join(', ')}"
```

Comment:
Would it be more readable if each was on a new line?

Reply:
Fair call. Done.